### PR TITLE
Add support for name parameter tests

### PR DESCRIPTION
**Pull request by wasabinza**
I have created a new pull request based on my copy of the develop branch rather than master.

### CODE REVIEW

**Comment by csoltenborn** [comment]
Ok... sorry for being not that responsive at the moment. I will soon be on vacation (and not be able to take care of the merge or your pull request before that), but will afterwards (from mid of September) tackle that stuff.

Just had quick look at the failing tests of the CI build - looks good, but we might need to adjust these numbers a bit (it's cumbersome, but easily fixed). Looking forward to review and then pull in your work!

**Comment by wasabinza** [comment]
No problem for the time scale, I'll hear from you when you are back.
The two failing tests in appveyor surprise me as I get the original numbers locally and they are significantly different and less, I don't know how I triggered the change.  If I look back through the history of appveyor it seems that those numbers were matching until I tried to sync with the master and then switch the existing branch to the develop branch, how that affects the current builds I don't understand.

**Comment by csoltenborn** [comment]
I might have fixed the failing tests - it looks like the number of found symbols varies between different compiler versions, so I decided to relax the test condition for those tests...

File: GoogleTestAdapter/Core.Tests/GoogleTestDiscovererTests.cs [hunk 1]
```diff
@@ -302,12 +302,12 @@ private void AssertIsGoogleTestExecutable(string executable, bool isGoogleTestEx
                 .Be(isGoogleTestExecutable);
         }
 
-        private void FindTests(string location)
+        private void FindTests(string location, int? expectedNrOfTestCases = null)
```
Comment (csoltenborn) [inline GoogleTestAdapter/Core.Tests/GoogleTestDiscovererTests.cs]: Using a plain `int` parameter with default value `TestResources.NrOfTests` is slightly clearer imho

Comment (wasabinza) [reply GoogleTestAdapter/Core.Tests/GoogleTestDiscovererTests.cs]: Yes you are right.  I have tidied this up, it is much clearer and less code.

**Comment by csoltenborn** [comment]
As a side note (and in case you haven't noticed already): You can download the VSIX created from your pull request [here](https://ci.appveyor.com/project/csoltenborn/googletestadapter-u1cxh/build/0.10.2.878/job/h2ha7dxvrmkipy3k/artifacts) - I thought that this version might be useful for you since you are apparently using that gtest feature.

**Pull request merged**